README.md

```diff
@@ -187,6 +187,7 @@ _Related: [Internet of Things (IoT)](#internet-of-things-iot)_
 - [Podgrab](https://github.com/akhilrex/podgrab) - Lightweight podcast manager and automatic podcast episode downloader. It will monitor podcasts for your and download them automatically whenever a new episode goes live. `GPL-3.0` `Docker/Go`
 - [pyLoad](https://pyload.net/) - Lightweight, customizable and remotely manageable downloader for 1-click-hosting sites like rapidshare.com or uploaded.to. ([Source Code](https://github.com/pyload/pyload)) `GPL-3.0` `Python`
 - [Radarr](https://radarr.video/) - Radarr is an independent fork of Sonarr reworked for automatically downloading movies via Usenet and BitTorrent, à la Couchpotato. ([Source Code](https://github.com/Radarr/Radarr)) `GPL-3.0` `C#`
+- [Redactics](https://www.redactics.com) `⚠` - The Redactics Agent software is a managed appliance (with free developer options) self-installed next to your databases that powers a growing collection of data management workflows. ([Demo](https://app.redactics.com))
```

Comment:
- Missing license information
- Missing language/platform information
- Missing source code link

Please refer to contribution guidelines above.

Reply:
Thank you!

It was my understanding that free self-hosted (but non-open source) stuff was fair game for this repo, but we are actually releasing our open source edition in a matter of days. If you don't mind my keeping this PR open for a little longer, I'll update this once we've launched!